src/main/scala/com/stackstate/pac4j/AkkaWebContextUtils.scala

```diff
@@ -0,0 +1,13 @@
+package com.stackstate.pac4j
+
+import org.pac4j.core.context.WebContext
+
+object AkkaWebContextUtils {
+
+  def withAkkaContext[A](context: WebContext)(f: AkkaHttpWebContext => A): A = {
+    context match {
+      case akka: AkkaHttpWebContext => f(akka)
+      case _ => throw new UnsupportedOperationException("Non Akka WebContext is not supported")
```

Comment:
Can we rephrase, please? Isn't that sentence confusing in a log mess?
```suggestion
      case _ => throw new UnsupportedOperationException("Only Akka Http WebContext is actually supported")
```